home-application/src/main/java/com/softserveinc/ita/homeproject/config/DataLoader.java

```diff
@@ -0,0 +1,65 @@
+package com.softserveinc.ita.homeproject.config;
```

Comment:
Please add the module name identity to the package.

com.softserveinc.ita.homeproject.application.config;